File: woodroid/src/main/java/me/gilo/woodroid/models/filters/WebhookFilter.java
package me.gilo.woodroid.models.filters;

public class WebhookFilter extends ListFilter{

    String status;

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
        addFilter("status", status);
    }
}

File: core/src/main/java/me/gilo/woodroid/core/models/filters/WebhookFilter.kt
package me.gilo.woodroid.models.filters

class WebhookFilter : ListFilter() {

    internal lateinit var status: String

    fun getStatus(): String {
        return status
    }

    fun setStatus(status: String) {
        this.status = status
        addFilter("status", status)
    }
}
